acceptance/lib/helper.rb

```diff
@@ -25,7 +25,7 @@ def self.initialize_config(options)
 
     puppet_version = get_option_value(options[:puppet_version],
                          nil, "Puppet Version", "PUPPET_VERSION",
-                         "4.99.0.31.g17aa7d8",
+                         "4.99.0-125-gf9a1b77",
```

Comment:
I'm not sure where exactly this is used, but the output when I run `! passing puppet-agent master` gives it with `.` rather than `-`....